core/src/test/scala/unit/kafka/tools/StorageToolTest.scala

```diff
@@ -206,12 +206,20 @@ Found problem:
 
   @Test
   def testFormatSucceedsIfAllDirectoriesAreAvailable(): Unit = {
-    val availableDir1 = TestUtils.tempDir()
-    val availableDir2 = TestUtils.tempDir()
+    val availableDirs = Seq(TestUtils.tempDir(), TestUtils.tempDir(), TestUtils.tempDir()).map(dir => dir.toString)
     val stream = new ByteArrayOutputStream()
-    assertEquals(0, runFormatCommand(stream, Seq(availableDir1.toString, availableDir2.toString)))
-    assertTrue(stream.toString().contains("Formatting %s".format(availableDir1)))
-    assertTrue(stream.toString().contains("Formatting %s".format(availableDir2)))
+    assertEquals(0, runFormatCommand(stream, availableDirs))
```

Comment:
Maybe we can decorate it with following style
```scala
    val availableDirs = Seq(TestUtils.tempDir(), TestUtils.tempDir(), TestUtils.tempDir()).map(dir => dir.toString)
    val stream = new ByteArrayOutputStream()
    assertEquals(0, runFormatCommand(stream, availableDirs))
    val actual = stream.toString().split("\\r?\\n")
    val expected = availableDirs.map("Formatting %s".format(_))
    // The duplicate messages mean some dirs get formated repeatedly
    assertEquals(availableDirs.size, actual.size)
    actual.foreach(line => expected.exists(line.startsWith))
```

Reply:
I tweaked it a little bit, made a two-way guarantee
@chia7712